src/dfm/data/load_datasets.py

```diff
@@ -0,0 +1,217 @@
+
+from typing import Union, Iterable, List, Optional
+import os
+from pathlib import Path
+from functools import partial
+
+from datasets import (
+    load_dataset,
+    interleave_datasets,
+    DatasetDict,
+    IterableDatasetDict,
+    load_from_disk,
+    interleave_datasets,
+    concatenate_datasets
+)
+
+HOPETWITTER_PATH = Path("/work") / "twitter_cleaned"
+DAGW_DFM_PATH = Path("/work") / "dagw-clean"
+DANEWS_PATH = Path("/work") / "hope-infomedia_cleaned"
+NAT_PATH = Path("/work") / "netarkivet-cleaned"
+
+
+
+def __add_source(example, source):
+    example["source"] = source
+    return example
+
+def keep_columns(
+    dataset: Union[IterableDatasetDict, DatasetDict], 
+    columns_to_keep: List[str]
+) -> Union[IterableDatasetDict, DatasetDict]:
+    subset = dataset[list(dataset.keys())[0]]
+    sample = next(iter(subset))
+    col_to_remove = [c_name for c_name in sample.keys() if c_name not in columns_to_keep]
+    
+    return dataset.remove_columns(col_to_remove)
+
+
+
+def load_hopetwitter(
+        path_to_hopetwitter: Union[str, Path]=HOPETWITTER_PATH, 
+        columns_to_keep: Optional[List[str]]=None, 
+        repeats: int=1,
```

Comment:
```suggestion
        n_training_repeats: int=1,
```
I'd consider a more informative arg-name like the above. Also missing in docstring – same for all the other functions.

Reply:
I agree